### ISSUE TITLE
Grid edit causes misaligned columns

### ISSUE THREAD

**Issue by sl4315** [vaadin/framework#10998]
Vaadin Framework version: 7.7.9 and 7.7.13 
Browser: Google Chrome 67.0.3396.87

When scrolling in a grid horizontally then clicking into the grid to edit, the alignment with the other rows columns and header gets out of sync.

![gridissue](https://user-images.githubusercontent.com/388708/41890181-b3a8577e-7951-11e8-9107-91e76fcc8671.jpg)

Steps to reproduce: 
1)	In application with multiple columns (horizontal scrollbar) scroll to somewhere in the middle of the grid. (In sample app attached for instance “fooBar10”)
2)	Click into grid to edit (for instance cell with “fooBar10”) -> columns no longer aligned with the header or the other rows.

This is an issue in Vaadin 7.7.9 and also with version 7.7.13 (unfortunately not in a position to change to Vaadin 8).

Attached is a tiny sample application (Spring Boot + Gradle) which is used to reproduce the issue.
[GridIssueSampleApp.zip](https://github.com/vaadin/framework/files/2135623/GridIssueSampleApp.zip)

**Comment by anasmi** [comment]
Reproducible in simple Vaadin application. Looks like the problem is that the left property of _v-grid-editor-cells_ is not calculated correctly(not always happens, but often), when row is scrolled. Underlying grid's row has the correct position, but editor row is misplaced. Clicking in another cell of the Grid editor makes row to be aligned correctly (left property is recalculated)

Simple example to reproduce:  
```
        // Create a grid
        Grid grid = new Grid();
        grid.setEditorEnabled(true);
        grid.setEditorBuffered(false);

        // Define some columns
        grid.addColumn("name", String.class);
        grid.addColumn("born", Integer.class);

        grid.addColumn("name1", String.class);
        grid.addColumn("born1", Integer.class);

        grid.addColumn("name2", String.class);
        grid.addColumn("born2", Integer.class);

        grid.addColumn("name3", String.class);
        grid.addColumn("born3", Integer.class);

        grid.addColumn("name4", String.class);
        grid.addColumn("born4", Integer.class);

        grid.setWidth("450px");

        // Add some data rows
        grid.addRow("Nicolaus Copernicus", 1543,"Nicolaus Copernicus", 1543,"Nicolaus Copernicus", 1543,"Nicolaus Copernicus", 1543,"Nicolaus Copernicus", 1543);

        grid.addRow("Galileo Galilei", 1564,"Galileo Galilei", 1564, "Galileo Galilei", 1564 , "s", 55,"Nicolaus Copernicus", 1543);

        grid.addRow("Johannes Kepler", 1571,"Johannes Kepler", 1571,"Johannes Kepler", 1571,"Nicolaus Copernicus", 1543,"Nicolaus Copernicus", 1543);

        layout.addComponent(grid);
```
In the screenshot above reindeer theme is used, but problem reproduced also with Valo.

**Comment by TatuLund** [comment]
Fixed in 7.7.14
